include/kf/stl/new

```diff
@@ -2,31 +2,33 @@
 
 #pragma warning(push)
 #pragma warning(disable : 4595) // non-member operator new or delete functions may not be declared inline
+#pragma warning(disable : 4996) // TODO: ExAllocatePoolWithTag is deprecated, use ExAllocatePool2
```

Comment:
Let's use suppress and move above `ExAllocatePoolWithTag`, so it will affect only one line.

Reply:
++